src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+const elements = document.querySelectorAll('span.population');
+const total = document.querySelector('span.total-population');
```

Comment:
Similarly, ensure that the 'total' and 'avgPop' elements are found before updating their innerText to prevent errors if they are missing.